### DISCUSSION TITLE
Why does Espanso use up 2-400MB of RAM?!

### DISCUSSION THREAD

**Discussion by YousufSSyed** [espanso/espanso#1914] [Q&A] [answered]
For something as seeemingly simple as this it shouldn't take up such a large chunk, I don't use the text expansion often either. I'm on an M1 MBP with 16GB of RAM.

**Comment by smeech** [comment]
![image](https://github.com/espanso/espanso/assets/47302288/2c22102c-da74-4214-80ec-e9769437bf3c)
About 30MB here.

**Comment by federico-terzi** [comment, marked as answer]
@YousufSSyed It's probably due to this memory leak on macOS https://github.com/espanso/espanso/issues/1675#issuecomment-1676366485
No solution available at the moment, but restarting Espanso is a possible workaround
